src/S3/MultipartCopy.php

```diff
@@ -51,24 +52,25 @@ class MultipartCopy extends AbstractUploadManager
      *   result of executing a HeadObject command on the copy source.
      *
      * @param S3ClientInterface $client Client used for the upload.
-     * @param string            $source Location of the data to be copied
+     * @param string $source Location of the data to be copied
      *                                  (in the form /<bucket>/<key>).
-     * @param array             $config Configuration used to perform the upload.
+     * @param array $config Configuration used to perform the upload.
      */
     public function __construct(
         S3ClientInterface $client,
         $source,
         array $config = []
-    ) {
+    )
+    {
```

Comment:
Style nitpick: `) {` on the same line is the convention we use in the SDK for multiline declarations